scripts/cdn_publish.sh

```diff
@@ -1,26 +1,38 @@
 #!/bin/bash
 cdnGravityDir="gravity-ui-web"
 cdnDir="cdn-dist"
-allVersions=`git tag --list --sort=-committerdate | grep -E '^(gravity-ui-web-)?v[0-9]*.[0-9]*.[0-9]*'`
 
-# First tag in the list will be the latest one, but may be prefixed (e.g. "gravity-ui-web-v1.2.3")
-version=`echo $allVersions | cut -d" " -f1`
-# be sure we get only the version part and discard any prefix
-version=`expr $version : '.*\(v[0-9].*\)'`
+if [ ! -z "$TRAVIS_COMMIT_RANGE"]
+then
+  # TRAVIS_COMMIT_RANGE has a bug that includes commits outside the PR, so fix with known workaround that is future safe
+  cdnChanges=`git diff --pretty="format:" --name-only ${TRAVIS_COMMIT_RANGE/.../..} | grep -E '^packages/gravity-ui-web/src*'`
```

Comment:
Should the grep (`grep -E '^packages/gravity-ui-web/src*'`) drop the `src` on the end? Otherwise changes to package.json won't cause a publish for example.